docs/getting-started.md

```diff
@@ -3,73 +3,97 @@
 
-3. Navigate to the Elasticsearch installation directory.
+## Configuration
 
-	```bash
-	$ cd /usr/share/elasticsearch
-	```
+Open the Elasticsearch configuration file (**/etc/elasticsearch/elasticsearch.yml**) and add the following to the end of the file.
 
-4. Download and install the plug-in package.
+```yaml
+couchbase.username: <USERNAME>
+couchbase.password: <PASSWORD>
+couchbase.maxConcurrentRequests: 1024
+```
 
-	```bash
-	$ bin/plugin install https://github.com/couchbaselabs/elasticsearch-transport-couchbase/releases/download/3.0.0-cypress/elasticsearch-transport-couchbase-3.0.0-alder-es2.4.6.zip
-	```
+The **username** and **password** credentials will be used again later when configuring Couchbase Server.
 
-When the installation is successful, the message "Installed transport-couchbase into /usr/share/elasticsearch/plugins/transport/couchbase" is logged.
+Still in the **/usr/share/elasticsearch** directory, configure the Elasticsearch plugin with the following curl command (Couchbase Server must be running).
 
-5. Open the Elasticsearch configuration file (**/etc/elasticsearch/elasticsearch.yml**) and add the following to the end of the file. The **username** and **password** should be the one used to connect to Couchbase Server.
+```bash
+$ curl -X PUT http://localhost:9200/_template/couchbase -d @plugins/transport-couchbase/couchbase_template.json
+```
 
-	```yaml
-	couchbase.username: <USERNAME>
-	couchbase.password: <PASSWORD>
-	couchbase.maxConcurrentRequests: 1024
-	```
+When successful, the configuration-routine provides the following response: `{"acknowledged":true}`.
 
-6. Start Elasticsearch
+Create an Elasticsearch index to receive the data from the Couchbase bucket. Later, when configuring XDCR, the "remote bucket name" must match the Elasticsearch index name.
 
-	```bash
-	$ /etc/init.d/elasticsearch start
-	```
+```bash
+$ curl -X PUT http://localhost:9200/travel-sample
 
-7. Still in the **/usr/share/elasticsearch** directory, configure the Elasticsearch plugin with the following curl command (Couchbase Server must be running).
+{"acknowledged":true}
+```
 
-	```bash
-	$ curl -X PUT http://localhost:9200/_template/couchbase -d @plugins/transport-couchbase/couchbase_template.json
-	```
+Now to configure Couchbase Server, open the Couchbase Web Console and select **XDCR > Add Remote Cluster**.
 
-	When successful, the configuration-routine provides the following response: `{"acknowledged":true}`.
+![](img/xdcrInitial.png)
 
-8. Instantiate an Elasticsearch index to be applied by Elasticsearch to the data in a given Couchbase bucket (in this case, the **travel-sample** bucket).
+In the dialog, enter the **Cluster Name** of your choice, the **IP/hostname** should correspond to the Elasticsearch cluster and **Username**/**Password** to the credentials previously stored in **elasticsearch.yml**.
```

Comment:
Might be good to emphasize the port number 9091 needs to be specified, since by default the ES plugin does not listen on the standard XDCR port.